charts/adot-exporter-for-eks-on-ec2/templates/aws-for-fluent-bit/configmap.yaml

```
@@ -127,7 +127,7 @@ data:
     [FILTER]
         Name                aws
         Match               dataplane.*
-        imds_version        v1
+        imds_version        {{ .Values.fluentbit.imds }}
```

```suggestion
        imds_version        {{ .Values.fluentbit.imdsVersion }}
```

A bit more specificity is probably good here.